ext/date/lib/interval.c

```diff
@@ -25,15 +25,15 @@
 #include "timelib.h"
 #include <math.h>
 
-timelib_rel_time *timelib_diff(timelib_time *one, timelib_time *two)
+timelib_rel_time *timelib_diff(timelib_time *one, timelib_time *two, bool absolute)
 {
 	timelib_rel_time *rt;
 	timelib_time *swp;
 	timelib_sll dst_corr = 0 ,dst_h_corr = 0, dst_m_corr = 0;
 	timelib_time one_backup, two_backup;
 
 	rt = timelib_rel_time_ctor();
-	rt->invert = 0;
+	rt->invert = absolute ? 1 : 0;
```

Comment:
If `absolute` has a boolean value, then is this branch really needed? `rt->invert = absolute;` should suffice.